cg/clients/janus/dto/create_qc_metrics_request.py

```diff
@@ -1,20 +1,7 @@
 """Post request model for JanusAPI."""
 
-from enum import StrEnum
-
 from pydantic import BaseModel
 
-from cg.constants import Workflow
-
-
-class SupportedWorkflow(StrEnum):
-    BALSAMIC: str = Workflow.BALSAMIC
-
-
-class SupportedPrepCategory(StrEnum):
-    TGA: str = "tga"
-    WGS: str = "wgs"
```

Comment:
will move this validation to Janus, cg should not need to care about this just receive errors